src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -123,7 +123,7 @@ const FileSystemConfiguration = () => {
           <div>
             <Alert
               isInline
-              variant="warning"
+              variant="danger"
```

Comment:
@regexowl , I define it as warning because Kelsea asked me to change it

Reply:
I see! Updated the PR, thanks!